skycatalogs/scripts/create_sc.py

```diff
@@ -57,10 +58,23 @@
                      Persist git provenance information for each file
                      written. Only supported format currently is as a
                      small yaml file, written to the data directory.''')
-
 parser.add_argument('--dc2', action='store_true', help='If supplied provide values comparable to those used for DC2 run. Default is False')
+parser.add_argument('--file', default='', help='''
```

Comment:
I would normally make the default `None`, but I guess an empty string is ok.

Also, is `file` really the best name for this?   Maybe something more descriptive like `options_file` would be more memorable.

Reply:
Yes, I've made this change